<?php

namespace Container5IEQoaE;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getDebug_Security_Firewall_Authenticator_NewApiShopUser_InnerService extends App_KernelTestDebugContainer
{
    /**
     * Gets the private 'debug.security.firewall.authenticator.new_api_shop_user.inner' shared service.
     *
     * @return \Symfony\Component\Security\Http\Firewall\AuthenticatorManagerListener
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/security-http/Firewall/AuthenticatorManagerListener.php';

        $a = ($container->privates['security.authenticator.manager.new_api_shop_user'] ?? $container->load('getSecurity_Authenticator_Manager_NewApiShopUserService'));

        if (isset($container->privates['debug.security.firewall.authenticator.new_api_shop_user.inner'])) {
            return $container->privates['debug.security.firewall.authenticator.new_api_shop_user.inner'];
        }

        return $container->privates['debug.security.firewall.authenticator.new_api_shop_user.inner'] = new \Symfony\Component\Security\Http\Firewall\AuthenticatorManagerListener($a);
    }
}
